app/styles/components/_c-interface.scss

```diff
@@ -13,10 +13,12 @@
   --sidebar: 100%;
 
   flex-basis: var(--sidebar);
+  // Without this `width`, the sidebar dissapears from the page on some breakpoints if the content is too wide
+  width: var(--sidebar);
```

Comment:
The issue was already there in master, but it's only reproducible in this PR because of the new select. So I included it here.

Very strange bug, I'm not sure if this is the best fix either, but it shall have to do for now 😄.